source/common/util/generate-id.ts

```diff
@@ -21,6 +21,6 @@ import replaceStringVariables from './replace-string-variables'
  *
  * @return  {String}                            The final string after replacements.
  */
-export default function generateId (pattern: string = '%Y%M%D%h%m%s'): string {
+export default function generateId (pattern: string = '%Y%M%D%h%m') {
```

Comment:
Please undo this change, you removed the (for this change unaffected) `%s`-variable, and removed the return type.